src/mdbExtensionController.ts

```diff
@@ -137,12 +137,14 @@ export default class MDBExtensionController implements vscode.Disposable {
   ): void => {
     const commandHandlerWithTelemetry = (args: any[]) => {
       // Send metrics to Segment
-      this._telemetryController?.track(
-        TelemetryEventTypes.EXTENSION_COMMAND_RUN,
-        {
-          command
-        }
-      );
+      if (this._telemetryController?.needTelemetry()) {
```

Comment:
I think because we check `if (this.needTelemetry()) {` in the telemetry controller we can remove this if?